City-of-Helsinki/drupal-hdbt-admin
# Fix sticky header on admin theme

How to test:

1. `composer create-project City-of-Helsinki/drupal-helfi-platform:dev-main hel-platform --no-interaction --repository https://repository.drupal.hel.ninja/`
2. Switch to correct branch: `composer require drupal/hdbt_admin:dev-UHF-X-admin-theme-fixes`
3. Run `make new`
4. Log in. Now when scrolling down in any page on admin side you should see the header being sticky on the top. This was broken before.
